tests/test_visitors/test_ast/test_keywords/test_raise.py

```diff
@@ -130,3 +131,22 @@ def test_bare_raise(
     visitor.run()
 
     assert_errors(visitor, [])
+
+
+def test_bare_raise_no_except(
```

Comment:
let's add tests (or even better one parametrized test) for following cases:

+ bare `raise` inside `except` block
+ `raise SomeException()` inside `except` block
+ `raise SomeException()`  not nested in `except` block